tools/Customization/DevHome.Customization/ViewModels/FileExplorerViewModel.cs

```diff
@@ -187,4 +212,54 @@ await Task.Run(async () =>
         });
         RefreshTrackedRepositories();
     }
+
+    public bool CalculateEnabled(string settingName)
+    {
+        if (LocalSettingsService!.HasSettingAsync(settingName).Result)
+        {
+            return LocalSettingsService.ReadSettingAsync<bool>(settingName).Result;
+        }
+
+        // Settings disabled by default
+        return false;
+    }
+
+    [RelayCommand]
+    public async Task OnToggledVersionControlIntegrationSettingAsync()
+    {
+        IsVersionControlIntegrationEnabled = !IsVersionControlIntegrationEnabled;
+        await LocalSettingsService!.SaveSettingAsync("VersionControlIntegration", IsVersionControlIntegrationEnabled);
+
+        if (!IsVersionControlIntegrationEnabled)
+        {
+            _log.Information("The user has disabled version control integration inside Dev Home");

```

Comment:
Why does the code log here, but not in the other two methods below?

Reply:
Yes, it seems like a good idea to add logging for the other methods as well. Modified in latest commit. Thanks!